pesticide/template_test_helpers.go

```diff
@@ -56,6 +57,9 @@ func TestTemplate(t *testing.T, registry sprout.Registry, tmplString string, dat
 func runTemplate(t *testing.T, handler sprout.Handler, tmplString string, data any) (string, error) {
 	t.Helper()
 
+	// Force timezone to UTC (for date parsing)
+	time.Local = time.UTC
```

Comment:
That's a dirty hack

Running this tests will randomly affects other tests depending on the order they are run.

At least you should use a t.Cleanup to restore the original value